lib/firespring_dev_commands/templates/git.rb

```diff
@@ -159,6 +159,49 @@ def create_merge_task!
           end
         end
       end
+
+      # Create the rake task for the git commit status task.
+      def create_commit_status_task!
+        # Have to set a local variable to be accessible inside of the instance_eval block
+        exclude = @exclude
+
+        DEV_COMMANDS_TOP_LEVEL.instance_eval do
+          namespace :git do
+            return if exclude.include?(:commit_status)
+
+            # TODO: Clean, comments and description
+            desc 'Add status to commit' \
+                 "\n\tuse TODO"
+
+            task :create_commit_status do
```

Comment:
I don't have an issue with this - just wanted to throw out some other potential options here:

1.) Passing in too many ENV options can make these commands difficult to use (definitely a balancing act) - you _could_ create task for each supported status? (e.g. `rake git:commit:status:success` or something). Then you wouldn't have to verify the status is in your list.

2.) You _could_ nest this under the application itself which would potentially get rid of the need for the `REPOSITORY` input (e.g. `rake accounts:git:commit:status:success` or something)

I haven't fully vetted those ideas and I don't feel strongly about them - just thoughts.

Reply:
I liked both ideas here, but our naming conventions are not currently one-to-one. I did pull the statuses out and made them independent.

The repo I would like to do, but I think we would add more then it removes. 

(name) springboard -> (repo) springboard
(name) accounts -> (repo) firespring-accounts
(name) sslapp -> (repo) firespring-ssl-management